picard/formats/id3.py

```diff
@@ -281,6 +281,12 @@ def _save(self, filename, metadata):
             tags.delall('APIC')
 
         encoding = {'utf-8': 3, 'utf-16': 1}.get(config.setting['id3v2_encoding'], 0)
+        
+        def addTXXX(desc, values, sep=config.setting['id3v23_join_with']):
```

Comment:
Can you rename this function to snake case? see https://github.com/metabrainz/picard/blob/master/CONTRIBUTING.md#picard-specific-code